nodes/nodes.go

```diff
@@ -94,13 +105,28 @@ func (r *Report) Print() {
 
 // AddToReport adds the given node to the report
 func (r *Report) AddToReport(node Node) {
-	v, _ := node.Version()
+	var (
+		v, _      = node.Version()
+		badBlocks = node.BadBlocks()
```

Comment:
I think we should do this more seldom. Like maybe once per minute.

Reply:
Okay, retrieving the bad blocks is now done once a minute